requirements.txt

```diff
@@ -1,4 +0,0 @@
-requests==2.22.0
```

Comment:
dont we need this?

Reply:
```python
    install_requires=["requests==2.22.0",
                      "emoji==0.5.3",
                      "python-telegram-bot==12.2",
                      "redis==3.3.8"],
```
The specification in the setup.py seems to be a replacement. According to my tests the pinning etc. also works. The requirements.txt is completely ignored here, so it can be removed.